### PR TITLE
chore: update certs

### PR DESCRIPTION
**Pull request by ninocomputer**
## Description

Updating certs in adapter + adding the expiration date

## Other changes

-

## Tested

-

## Related issues

- Fixes https://github.com/mento-protocol/mento-general/issues/269

## Backwards compatibility

-

### CODE REVIEW

File: devPriceSourcesConfig.txt [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+      [
+        {exchange: 'COINBASE', symbol: 'EUROCUSD', toInvert: false},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: true},
+        {exchange: 'COINBASE', symbol: 'USDTEUR', toInvert: false}
+      ],      
+      [
+        {exchange: 'BITSTAMP', symbol: 'EUROCEUR', toInvert: false},
+      ]    
+    ]
+
```
Comment (bowd) [inline devPriceSourcesConfig.txt:14]: Revert this from the PR

**Review by bowd — CHANGES_REQUESTED** [review]

File: devPriceSourcesConfig.txt (hunk 1 above)
Comment (ninocomputer) [reply devPriceSourcesConfig.txt:14]: 👍🏼

File: src/exchange_adapters/binance_us.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { BinanceAdapter } from './binance'
 export class BinanceUSAdapter extends BinanceAdapter implements ExchangeAdapter {
   baseApiUrl = 'https://api.binance.us/api/v3'
   readonly _exchangeName = Exchange.BINANCEUS
-  // GeoTrust RSA CA 2018
+  // *.binance.com - validity not after: 17/02/2024, 00:59:59 CET
   readonly _certFingerprint256 =
-    '8C:C3:4E:11:C1:67:04:58:24:AD:E6:1C:49:07:A6:44:0E:DB:2C:43:98:E9:9C:11:2A:85:9D:66:1F:8E:2B:C7'
+    '93:07:DE:DD:AF:3A:78:77:1D:B1:B7:68:3E:9F:18:8E:28:83:AE:A1:77:58:87:D4:5C:F6:F9:C8:71:1A:72:49'
```
Comment (nvtaveras) [inline src/exchange_adapters/binance_us.ts:10]: this is the certificate + validity from binance.com, not binance.us

File: src/exchange_adapters/coinbase.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { CeloContract } from '@celo/contractkit'
 
 export class CoinbaseAdapter extends BaseExchangeAdapter implements ExchangeAdapter {
   baseApiUrl = 'https://api.pro.coinbase.com'
-  // Cloudflare Inc ECC CA-3
+  // pro.coinbase.com - validity not after: 11/05/2024, 01:59:59 CEST
```
Comment (nvtaveras) [inline src/exchange_adapters/coinbase.ts:8]: looks like you updated the validity here but not the certificate. For pro.coinbase.com I get `7B:F6....3B:39`

File: src/exchange_adapters/novadax.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { Exchange } from '../utils'
 export class NovaDaxAdapter extends BaseExchangeAdapter implements ExchangeAdapter {
   baseApiUrl = 'https://api.novadax.com/v1/market'
   readonly _exchangeName = Exchange.NOVADAX
-  // NovaDAX's certificate fingerprint.
+  // novadax.com - validity not after: 23/11/2023, 08:35:21 CET
```
Comment (nvtaveras) [inline src/exchange_adapters/novadax.ts:8]: This is the right validity but wrong certificate. I get `D9:77...5B:1D`

File: src/exchange_adapters/whitebit.ts [hunk 5]
```diff
@@ -5,9 +5,9 @@ import { Exchange } from '../utils'
 export class WhitebitAdapter extends BaseExchangeAdapter {
   baseApiUrl = 'https://whitebit.com/api/v4/public/'
   readonly _exchangeName = Exchange.WHITEBIT
-  // Cloudflare Inc ECC CA-3
+  // whitebit.com - validity not after: 19/09/2023, 01:59:59 CEST
```
Comment (nvtaveras) [inline src/exchange_adapters/whitebit.ts:8]: hmm.. this is expiring in 12 days already, I'd propose for this one we keep the old one and create an issue to check whitebit again in 2 weeks? To not update it not and then have it expire right after.

**Review by nvtaveras — CHANGES_REQUESTED** [review]

File: src/exchange_adapters/binance_us.ts (hunk 2 above)
Comment (ninocomputer) [reply src/exchange_adapters/binance_us.ts:10]: we use the same certificate for them

File: src/exchange_adapters/whitebit.ts (hunk 5 above)
Comment (ninocomputer) [reply src/exchange_adapters/whitebit.ts:8]: yeah sounds good

File: src/exchange_adapters/bitstamp.ts [hunk 6]
```diff
@@ -15,6 +15,7 @@ export class BitstampAdapter extends BaseExchangeAdapter implements ExchangeAdap
     const base = BitstampAdapter.tokenSymbolMap.get(this.config.baseCurrency)
     const quote = BitstampAdapter.tokenSymbolMap.get(this.config.quoteCurrency)
     return `${base}${quote}`.toLowerCase()
+    return `${base}${quote}`.toLowerCase()
```
Comment (nvtaveras) [inline src/exchange_adapters/bitstamp.ts:18]: revert

File: src/exchange_adapters/whitebit.ts [hunk 7]
```diff
@@ -5,9 +5,9 @@ import { Exchange } from '../utils'
 export class WhitebitAdapter extends BaseExchangeAdapter {
   baseApiUrl = 'https://whitebit.com/api/v4/public/'
   readonly _exchangeName = Exchange.WHITEBIT
-  // whitebit.com - validity not after: 19/09/2023, 01:59:59 CEST
+  // Cloudflare Inc ECC CA-3
```
Comment (nvtaveras) [inline src/exchange_adapters/whitebit.ts:8]: expiry date?

Comment (ninocomputer) [reply src/exchange_adapters/whitebit.ts:8]: we're changing it in 2 weeks

**Review by nvtaveras — APPROVED** [review]

**Review by bowd — APPROVED** [review]

**Pull request merged**